### PR TITLE
Feature/consolidate parsers

### PR DESCRIPTION
**Pull request by jkrasting**
- standalone PR for a common CLI parser.

### CODE REVIEW

File: om4labs/om4parser.py [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+            type=str,
+            default=None,
+            help="Path to ocean.static.nc file",
+        )
+
+    # intake catalog-related options
+    if "configuration" not in exclude:
+        parser.add_argument(
+            "-c",
+            "--configuration",
```
Comment (raphaeldussin) [inline om4labs/om4parser.py:193]: --config? a bit shorter

**Review by raphaeldussin — CHANGES_REQUESTED** [review]
minor changes and good to go

File: om4labs/om4parser.py [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    -g, --gridspec:     path;  gridspec tarfile
+    -s, --static:       path;  ocean.static dataset
+
+    Intake catalogs:
+
+    Intake catalogs will be used to specify default grid files for known model
+    configurations and for paths to observational datasets.  These will be the
+    preferred paths:
+
+    -m, --model:        str; model configuration (e.g. OM4p25, OM4p5, OM4p125)
```
[the remark this answers is not in the document]
Comment (jkrasting) [reply om4labs/om4parser.py]: Addressed in 480f9bf

File: om4labs/om4parser.py (hunk 1 above)
Comment (jkrasting) [reply om4labs/om4parser.py]: Addressed in 480f9bf

**Pull request merged**